lmnet/executor/train.py

```diff
@@ -40,7 +40,8 @@ def setup_dataset(config, subset, rank):
     dataset_kwargs = dict((key.lower(), val) for key, val in config.DATASET.items())
     dataset = DatasetClass(subset=subset, **dataset_kwargs)
     enable_prefetch = dataset_kwargs.pop("enable_prefetch", False)
-    return DatasetIterator(dataset, seed=rank, enable_prefetch=enable_prefetch)
+    cached_batch_count = config.DATASET.BATCH_PRE_FETCH
```

Comment:
Please use the same variable name, not `BATCH_PRE_FETCH` or `cached_batch_count`.

Reply:
@ruimashita San: Fixed in the next commit